src/libraries/illink-sharedframework.targets

```diff
@@ -70,6 +59,29 @@
       <ILLinkArgs>$(ILLinkArgs) -r @(_AssembliesToLink->'%(Identity)', ' -r ')</ILLinkArgs>
     </PropertyGroup>
 
+    <PropertyGroup>
+      <ProjectILLinkSuppressionsFile>src\ILLink\ILLink.Suppressions</ProjectILLinkSuppressionsFile>
+      <CoreLibSharedILLinkSuppressionsFilesPrefix>$(CoreLibSharedDir)ILLink\ILLink.Suppressions.Shared</CoreLibSharedILLinkSuppressionsFilesPrefix>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <!-- Include suppression XML files bin-placed in earlier per-library linker run. -->
+      <_SuppressionsXmls Include="$(ILLinkTrimAssemblySuppressionsXmlsDir)*.xml" />
+
+      <!-- Collate CoreLib suppression XML files not bin-placed in earlier per-library linker run. CoreLib doesn't use bin-place logic. -->
+      <_SuppressionsXmls Include="$(CoreLibSharedILLinkSuppressionsFilesPrefix).xml" />
+      <_SuppressionsXmls Condition="'$(TargetOs)' != 'Windows_NT'" Include="$(CoreLibSharedILLinkSuppressionsFilesPrefix).NonWindows.xml" />
+      <_SuppressionsXmls Condition="'$(RuntimeFlavor)' == 'Mono'" Include="$(MonoProjectRoot)netcore\System.Private.CoreLib\$(ProjectILLinkSuppressionsFile).xml" />
+
+      <!-- netstandard2.1-only System.ComponentModel.Annotations XML files are not bin-placed during -allConfigurations builds. -->
```

Comment:
Is this a bug/TODO?

Reply:
No, we have to due this. I've updated the comment to make it clearer.

```
<!-- netstandard2.1-only System.ComponentModel.Annotations XML file is not bin-placed during -allConfigurations builds
     due to the linker not being run on the assembly (eng\illink.targets). Manually include it. -->
```